.github/workflows/vuln-scans.yaml

```diff
@@ -0,0 +1,41 @@
+name: Vulnerability scans
+on:
+  workflow_dispatch: {}   # Allow for manual triggers
+  schedule:
+    - cron:  '0 8 * * *'  # Daily, at 8:00 UTC
+  pull_request:
+
+jobs:
+  trivy-scan:
+    name: Trivy security scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy scan on repo
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          format: 'table'
+          exit-code: '1'
+          ignore-unfixed: true
+          severity: 'CRITICAL,HIGH'
+
+      - name: Build image
+        run: |
+          make image
```

Comment:
Since this is run nightly, we can scan the latest published image, which is `openpolicyagent/opa:edge`

Reply:
Maybe I should just limit this workflow to run on PRs, and then add the `openpolicyagent/opa:edge` scan to the nightly run.